include/swift/AST/DiagnosticsSema.def

```diff
@@ -3831,7 +3831,8 @@ ERROR(decl_from_hidden_module,none,
       "in an extension with public or '@usableFromInline' members|"
       "in an extension with conditional conformances|"
       "in a public or '@usableFromInline' conformance|"
-      "in an '@available' attribute here}1; "
+      "in an '@available' attribute here|"
+      "in a variable declaration marked public or in a '@frozen' or '@usableFromInline' context}1; "
```

Comment:
I think we typically refer to these as properties in diagnostics, since it's not a local variable